/**
 */
package org.gemoc.xcapella.dataflow.xdsml.dataflowmt.information.communication;


/**
 * <!-- begin-user-doc -->
 * A representation of the model object '<em><b>Exception</b></em>'.
 * <!-- end-user-doc -->
 *
 *
 * @see org.gemoc.xcapella.dataflow.xdsml.dataflowmt.information.communication.CommunicationPackage#getException()
 * @model annotation="http://www.polarsys.org/capella/2007/BusinessInformation Label='Exception'"
 *        annotation="http://www.polarsys.org/capella/2007/UML2Mapping metaclass='Class' stereotype='eng.Exception'"
 *        annotation="http://www.polarsys.org/kitalpha/ecore/documentation description='A piece of  information raised (typically by an operation) to mention the occurence of an abnormal condition\r\n[source: Capella study]' usage\040guideline='n/a' used\040in\040levels='system,logical,physical' usage\040examples='n/a' constraints='none' comment/notes='not used/implemented as of Capella 1.0.3' reference\040documentation='none'"
 *        annotation="http://www.polarsys.org/capella/MNoE/CapellaLike/Mapping UML/SysML\040semantic\040equivalences='' base\040metaclass\040in\040UML/SysML\040profile\040='uml::Class' explanation='none' constraints='none'"
 * @generated
 */
public interface Exception extends CommunicationItem {
} // Exception
